### PR TITLE
Folder removal

### PR DESCRIPTION
**Pull request by cvaldez98**
On quit, Old joe will now clean all the things!
image

### CODE REVIEW

File: src/helpers/clean-restore.ts [hunk 1]
```diff
@@ -59,9 +59,12 @@ export async function deleteOldJoeFolders() {
     f => f.startsWith(`${USER_DATA_FOLDER_NAME}-`) && f.length === 22
   );
 
-  for (const folder of slackBackupFolders) {
-    const folderPath = path.join(appDataDir, folder);
-    if (hasOldJoeFile(folderPath)) {
+  if(slackBackupFolders != undefined && slackBackupFolders[0] != undefined) {
```
Comment (felixrieseberg) [inline src/helpers/clean-restore.ts]: Do me a favor and run `npm run prettier` 🙇

Comment (felixrieseberg) [inline src/helpers/clean-restore.ts]:
```suggestion
  if (slackBackupFolders && slackBackupFolders[0]) {
```

File: src/main/main.ts [hunk 2]
```diff
@@ -30,6 +32,7 @@ export function onBeforeQuit() {
 export function onWindowsAllClosed() {
   // On OS X it is common for applications and their menu bar
   // to stay active until the user quits explicitly with Cmd + Q
+  deleteOldJoeFolders();
```
Comment (felixrieseberg) [inline src/main/main.ts]: The `onbeforequit` above should be enough to handle this case. Is it not?

Comment (cvaldez98) [reply src/main/main.ts]: correct, deleted the line

**Pull request merged**